index.html

```diff
@@ -5,8 +5,121 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="main-header">
+      <nav class="main-navigation">
+        <ul class ="main-navigation-list">
+          <li class="main-navigation-item"><a href="info.html">Информация</a></li>
+          <li class="main-navigation-item"><a href="photos.html">Фото и видео</a></li>
+          <li class="main-navigation-item"><a href="map.html">Карта штата</a></li>
+          <li class="main-navigation-item"><a href="hotels.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+        <h1>Sedona</h1>
+        <section class="welcome">
+          <h2>Welcome</h2>
+            <span class="direction">to the gorgeous</span>
+            <p class="quote">"Because the Grand Canyon sucks!"</p>
+            <p class="main-description">Седона - небольшой городок в Аризоне, заслуживающий большего!</p>
+            <p class="reasons-for-visit">Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>
+        </section>
```

Comment:
Это 2 разных блока.
1 будет блок картинок с надписью седона; welcome; because
2 блок 
<p class="main-description">Седона - небольшой городок в Аризоне, заслуживающий большего!</p>
<p class="reasons-for-visit">Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>